sdk/core/azure-core/inc/azure/core/operation.hpp

```diff
@@ -105,6 +105,7 @@ namespace Azure { namespace Core {
 
     /**
      * @brief Returns the current #Azure::Core::OperationStatus of the long-running operation.
+     *
```

Comment:
Out of curiosity, what regex did you use?

Reply:
`/\*\*[\n\s]+.*[\n\s]+\*/`
/\*\*[\n\s]  ->  finds `/** then-whatever up to end of line
+ -> adds multiline support to check next line
.*  -> whatever on this line up to end of line
*/  ->  end of doxygen..

So, any doxygen comment with one line length only :)